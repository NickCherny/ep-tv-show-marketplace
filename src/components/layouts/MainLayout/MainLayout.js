import React from 'react';
import AppBar from "@material-ui/core/AppBar";
import Toolbar from '@material-ui/core/Toolbar';
import Container from '@material-ui/core/Container';

const MainLayout = ({ children }) => {
  return (
    <main>
      <AppBar position="relative">
        <Toolbar>
          Hmm...
        </Toolbar>
      </AppBar>
      <Container maxWidth="md">
        <article>
          {children}
        </article>
      </Container>
    </main>
  )
};

export default MainLayout;
