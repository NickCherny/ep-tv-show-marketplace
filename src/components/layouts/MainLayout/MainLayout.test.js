import React from 'react';
import renderer from 'react-test-renderer';

import MainLayout from "./MainLayout";

describe('<MainLayout />', () => {
  test('snapshot', () => {
    const tree = renderer.create(<MainLayout/>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
