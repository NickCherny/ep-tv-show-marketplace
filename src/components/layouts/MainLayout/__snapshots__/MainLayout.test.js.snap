// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MainLayout /> snapshot 1`] = `
<main>
  <header
    className="MuiPaper-root MuiPaper-elevation4 MuiAppBar-root MuiAppBar-positionRelative MuiAppBar-colorPrimary"
  >
    <div
      className="MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters"
    >
      Hmm...
    </div>
  </header>
  <div
    className="MuiContainer-root MuiContainer-maxWidthMd"
  >
    <article />
  </div>
</main>
`;
